vms/event/forms.py

```diff
@@ -16,7 +18,12 @@ class Meta:
             'state',
             'city',
             'address',
+<<<<<<< HEAD
```

Comment:
Merge conflict. Please remove unneeded lines such as ``<<<<<<<<<< HEAD`` and others related to the merge conflict.